.banzaicloud/charts/spotguide-spark/templates/NOTES.txt

```diff
@@ -87,17 +87,22 @@ echo '<copied values>' | base64 -D
 bin/spark-submit \
 --master k8s://<<kubernetes master ip>> \
 --deploy-mode cluster \
---class org.apache.spark.examples.SparkPi \
---conf spark.app.name=spark-pi \
---conf spark.kubernetes.driver.label.app=spark-pi \
---conf spark.kubernetes.executor.label.app=spark-pi \
+--conf spark.app.name=spark-example \
+--conf spark.kubernetes.driver.label.app=spark-example \
+--conf spark.kubernetes.executor.label.app=spark-example \
 --conf spark.kubernetes.authenticate.driver.serviceAccountName=spark \
 --conf spark.local.dir=/tmp/spark-locals \
 --conf spark.kubernetes.docker.image.pullPolicy=Always \
---conf spark.kubernetes.container.image={{ .Values.banzaicloud.spark.image.repository }}:{{ .Values.banzaicloud.spark.image.tag }} \
+--conf spark.kubernetes.container.image={{ template "sparkImageName" . }} \
+--conf spark.kubernetes.namespace={{ .Release.Namespace }} \
 --conf spark.kubernetes.authenticate.submission.caCertFile=<your path to certificate-authority-data.pem> \
```

Comment:
These are not needed as spark-submit favors kubeconfig. If there is a kubeconfig is available than the caCertFile, clientKeyFile, clientCertFile won't have any effect. The caCertFile, clientKeyFile, clientCertFile configs are used only if there is no kubeconfig available.

Reply:
removed